app/javascript/components/forms/SigninForm.jsx

```diff
@@ -35,11 +35,9 @@ export default function SigninForm() {
   return (
     <Form
       methods={methods}
-      onSubmit={(data) => {
-        captchaRef.current.execute({ async: true })
```

Comment:
I used the await because of another issue.

If you notice, the spinner on the `signin` and the `signup` button is no longer working.

The problem is due to not returning a promise on the `onSubmit` also.

Meaning that to fix it with the "thenable" syntax we'll have two stages of chaining piping the promise which a bit ugly then a double consecutive `await`.

Haven't checked if returning the `onSubmit` will make the `react-hook-form` itself `await` for its response asynchronously changing its state therefore working as expected.

But, IMO in this case the `async/await` will simplify the not needed complexity.

Reply:
Ah I see - makes sense